lib/reporters/tap.js

```diff
@@ -7,7 +7,7 @@ function getSourceFromStack(stack, index) {
 		.split('\n')
 		.slice(index, index + 1)
 		.join('')
-		.replace(/^\s+at /, '');
```

Comment:
And you removed `at` from the stack trace in a test: https://github.com/sindresorhus/ava/pull/570/files#diff-373bdde32276db897f685ce9d49e3b2cR63. But all "standard" stack traces have `at` at the beginning. That's why I was wondering whether we work with an already cleaned up stack here (by `stack-utils`).